Workbooks/Virtual Machines - Performance Analysis/Performance Analysis for a Single VM/Performance Analysis for a Single VM.workbook

```diff
@@ -159,7 +163,8 @@
             "query": "Perf\r\n| where TimeGenerated {TimeRange}\r\n| summarize by CounterName, ObjectName, CounterText = strcat(ObjectName, ' > ', CounterName)\r\n| order by ObjectName asc, CounterText asc\r\n| project Counter = pack('counter', CounterName, 'object', ObjectName), CounterText",
             "isHiddenWhenLocked": false,
             "timeContextFromParameter": null,
-            "resourceType": "microsoft.compute/virtualmachines"
+            "resourceType": "microsoft.compute/virtualmachines",
+            "value": "{\"counter\":\"% Used Inodes\",\"object\":\"Logical Disk\"}"
```

Comment:
If there is no data to display, this will display an empty chart. Can you please add a No data message and display that when there is no data instead of empty chart.

Something like this but with appropriate computer name:
![image](https://user-images.githubusercontent.com/45012871/51336351-25bb4a00-1a39-11e9-86ca-18c47a8d644b.png)


PS: Also check if this is one of the important counters, we can probably use a more widely used counter.